components/ActivitiesList/index.js

```diff
@@ -1,14 +1,10 @@
 import ActivityCardPreview from "../ActivityCardPreview";
 import { StyledUl, StyledListItem } from "../StyledList";
 
-export default function ActivitiesList({
-  activitiesInfo,
-  activities,
-  onToggleBookmark,
-}) {
+export default function ActivitiesList({ activitiesInfo, onToggleBookmark }) {
   return (
     <StyledUl>
-      {activities.map((activity) => (
+      {activitiesInfo.map((activity) => (
         <StyledListItem key={activity.id}>
           <ActivityCardPreview
             activity={activity}
```

Comment:
Since your activity already holds the information 'isBookmarked' you don't have to drill the prop 'isBookmarked'. In the 'ActivityCardPreview' component you can get the information from 'activity.isBookmarked'